config/pgoconfig.go

```diff
@@ -490,6 +495,13 @@ func (c *PgoConfig) Validate() error {
 			return errors.New(errPrefix + msg)
 		}
 	}
+
+	// if provided, ensure that the type of pod anti-affinity specified is valid
+	podAntiAffinityType := crv1.PodAntiAffinityType(c.Cluster.PodAntiAffinity)
+	if err := podAntiAffinityType.ValidatePodAntiAffinityType(); podAntiAffinityType != "" && err != nil {
```

Comment:
Shouldn't the validation have caught the `""` case here?

Reply:
If an empty affinity type is provided, then it will default to `preferred` once processed by the operator.  Therefore, since a empty value is not intended to be a affinity type, it does not need to be validated.  Instead, it simply indicates that the default should be applied (specifically by the Operator).